pylint/checkers/refactoring.py

```diff
@@ -261,13 +265,30 @@ def _check_redefined_argument_from_local(self, name_node):
                                  args=(name_node.name, ))
 
     @utils.check_messages('redefined-argument-from-local',
-                          'too-many-nested-blocks')
+                          'too-many-nested-blocks',
+                          'mutable-sequence-modified-in-loop')
     def visit_for(self, node):
         self._check_nested_blocks(node)
 
         for name in node.target.nodes_of_class(astroid.AssignName):
             self._check_redefined_argument_from_local(name)
 
+        inferrend = utils.safe_infer(node.iter)
```

Comment:
:'( Sorry for my English.
When inferred is None then enters in this case `inferred is astroid.util.Uninferable`